README.md

```diff
@@ -289,13 +433,13 @@ In current context, dependencies such as `react-native-document-picker` and (if
 
 ## Common issues
 
-#### While running native example, you may (not necessarily) run into the following issues:
+#### While running native example, you may (not necessarily) run into the following issues
 
-1. When you execute `react-native run-ios` for the first time, it starts a metro bundler in parallel. It can result in some errors since the build process isn't complete yet. Try the following to fix this:
+1. When you execute `npx react-native run-ios` for the first time, it starts a metro bundler in parallel. It can result in some errors since the build process isn't complete yet. Try the following to fix this:
```

Comment:
```suggestion
1. When you execute `yarn ios` for the first time, it starts a metro bundler in parallel. It can result in some errors since the build process isn't complete yet. Try the following to fix this:
```